src/compiler/checker.ts

```diff
@@ -10965,7 +10981,13 @@ module ts {
 
         function getExportNameSubstitution(symbol: Symbol, location: Node): string {
             if (isExternalModuleSymbol(symbol.parent)) {
-                return "exports." + unescapeIdentifier(symbol.name);
+                var symbolName = unescapeIdentifier(symbol.name);
+                if (languageVersion >= ScriptTarget.ES6) {
+                    return symbolName;

```

Comment:
can you add a comment why we don't prefix export name with 'exports.' when  emitting for ES6+?